osf/models/spam.py

```diff
@@ -200,7 +200,6 @@ def do_check_spam(self, author, author_email, content, request_headers):
             'user_agent': request_headers.get('User-Agent'),
             'referer': request_headers.get('Referer'),
         }
-        request_kwargs.update(request_headers)
```

Comment:
can you elaborate on this change, please?

Reply:
It's not mine. It appeared when I merge develop branch

Reply:
Yep. This was was a hotfix to avoid some breakage in some spam checks.  If this is still being applied, you may want to rebase against current develop.